api/mock/facade.go

```diff
@@ -12,18 +12,38 @@ import (
 
 // Facade is the mock implementation of a node router handler
 type Facade struct {
-	Running                                        bool
-	ShouldErrorStart                               bool
-	ShouldErrorStop                                bool
-	GetCurrentPublicKeyHandler                     func() string
-	TpsBenchmarkHandler                            func() *statistics.TpsBenchmark
-	GetHeartbeatsHandler                           func() ([]heartbeat.PubKeyHeartbeat, error)
-	BalanceHandler                                 func(string) (*big.Int, error)
-	GetAccountHandler                              func(address string) (*state.Account, error)
-	GenerateTransactionHandler                     func(sender string, receiver string, value *big.Int, code string) (*transaction.Transaction, error)
-	GetTransactionHandler                          func(hash string) (*transaction.Transaction, error)
-	SendTransactionHandler                         func(nonce uint64, sender string, receiver string, value *big.Int, gasPrice uint64, gasLimit uint64, code string, signature []byte) (string, error)
-	SendBulkTransactionsHandler                    func(txs []*transaction.Transaction) error
+	Running                    bool
+	ShouldErrorStart           bool
+	ShouldErrorStop            bool
+	GetCurrentPublicKeyHandler func() string
+	TpsBenchmarkHandler        func() *statistics.TpsBenchmark
+	GetHeartbeatsHandler       func() ([]heartbeat.PubKeyHeartbeat, error)
+	BalanceHandler             func(string) (*big.Int, error)
+	GetAccountHandler          func(address string) (*state.Account, error)
+	GenerateTransactionHandler func(sender string, receiver string, value *big.Int, code string) (*transaction.Transaction, error)
+	GetTransactionHandler      func(hash string) (*transaction.Transaction, error)
+	SendTransactionHandler     func(
```

Comment:
I would not apply in this very case the rule that states: each parameter on a new line. It is very confusing when working with func pointers

Reply:
done